build.gradle

```diff
@@ -1,17 +1,60 @@
 buildscript {
+    ext.versions = [
+            'compileSdk'                : 28,
+            'minSdk'                    : 14,
+            'targetSdk'                 : 28,
+            'versionName'               : VERSION_NAME,
+            'versionCode'               : VERSION_CODE,
+
+            'appcompat'                 : '28.0.0-rc02',
```

Comment:
I don't want to release on a rc version dependency

Reply:
change to 27.1.1